testsuite/tests/kuadrant/authorino/identity/anonymous/test_anonymous_context.py

```diff
@@ -0,0 +1,25 @@
+"""Test for anonymous identity context"""
+import json
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization):
+    """Setup AuthConfig for test"""
+    authorization.add_anonymous_identity("anonymous")
+    authorization.add_response({"name": "auth-json", "json": {
+        "properties": [{"name": "auth", "valueFrom": {"authJSON": "auth"}},
+                       {"name": "context", "valueFrom": {"authJSON": "context"}}]}})
```

Comment:
You should add only the things you need e.g. `{"name": "auth", "valueFrom": {"authJSON": "auth.identity.anonymous"}`